core/src/com/fallenflame/game/PlayerModel.java

```diff
@@ -11,10 +11,19 @@
 public class PlayerModel extends CharacterModel {
     /** Radius of player's light */
     protected float lightRadius = 0;
+    protected float forceSneak = 0;
+    protected float forceSprint = 0;
+    protected float forceWalk = 0;
```

Comment:
I don't think we need to save these and I don't think we need setters for these. We can just have getters like so:
```    /**
     * Gets player force for sneaking
     * @return player force for sneaking
     */
    public float getForceSneak() {
        return getForce() / 2;
    }

Reply:
Also lightRadiusSprint doesn't need a setter either. It can just be saved directly to the variable during initialization,